elements/campaign-display/campaign-display.test.js

```diff
@@ -14,6 +14,11 @@ describe('<campaign-display>', () => {
       campaignUrl,
       display: 'image',
     };
+
+    // Prevent setState warnings spamming the console
+    // We sould investigate if this is an issue with lib/bulbs-elements/store/store.js:60
```

Comment:
Put a `TODO` here to mark it a bit more clearly as something to investigate later.